src/js/dex/orderbook.component.js

```diff
@@ -38,6 +48,10 @@
     function OrderbookController() {
         var ctrl = this;
 
+        ctrl.cancel = function (obj) {
+            ctrl.cancelOrder(obj.order);
+        };
+
         ctrl.$onChanges = function () {
             if (ctrl.type !== 'user') {
```

Comment:
$onChanges is called gazillion times. It's better to check that property you need is changed and then apply some logic.

Hint: $onChanges accepts an argument that holds data about changed bindings:
ctrl.$onChanges = function (changesObj) {...}